src/fctGraph/FCTGraph.js

```diff
@@ -62,7 +62,7 @@ class FCTGraph {
   constructor({
     id,
     deviceId,
-    functionalities: functionalitiesData,
+    functionalities: functionalitiesData = [],
```

Comment:
this is new for me and cool. makes sense, but definitely did not know you could rename and assign a default value to a renamed argument